app/app.R

```diff
@@ -20,23 +20,54 @@ ui <- navbarPage(
     windowTitle = apptitle,
     theme = "styling.css",
     position="fixed-top", collapsible = TRUE,
-    tabPanel("Results", sidebarLayout(position = "right",
+    tabPanel("Results", sidebarLayout(position = "left",
       sidebarPanel(
-        numericInput("R",
-          "R:",
-          min = 0.5,
-          max = 10,
-          value = 2
+        dateInput("admissionDate",
+          "Date of admission:",
+          format = "dd/mm/yyyy"
```

Comment:
Let us use the [ISO format](https://en.wikipedia.org/wiki/ISO_8601) `yyyy-mm-dd`